### PR TITLE
Usecases

### PR DESCRIPTION
**Pull request by AtefehKashani**
- updated LA parameters
- updated the parameters in documentation and new images and added the video files for pca
- changed the prepoutput and pointfiles to groomed and shape_models to be consistent with the data portal.
- changed the ellipsoid outputs file to be consistent with the other use cases issue #614

### CODE REVIEW

**Comment by akenmorris** [comment]
Running now...

File: Examples/Python/ellipsoid.py [hunk 1]
```diff
@@ -88,28 +88,28 @@ def Run_Pipeline(args):
     if int(args.interactive) != 0:
         input("Press Enter to continue")
 
-    parentDir = 'TestEllipsoids/PrepOutput/'
+    parentDir = 'TestEllipsoids/groomed/'
```
Comment (akenmorris) [inline Examples/Python/ellipsoid.py:91]: Do we have the wrong zip file?  On a clean system (GitHub Action), this is failing.

@AtefehKashani @0leks

Comment (akenmorris) [reply Examples/Python/ellipsoid.py:91]: I guess this isn't the right place.  But in any case, the automated test doesn't work while looking for some file.

Comment (akenmorris) [reply Examples/Python/ellipsoid.py:91]: Ok, it was the path that the test was looking for output.

Comment (AtefehKashani) [reply Examples/Python/ellipsoid.py:91]: @akenmorris  Sorry I missed you r massages earlier. Did this solve the problem?

Comment (akenmorris) [reply Examples/Python/ellipsoid.py:91]: I committed the fix.

**Comment by akenmorris** [comment]
I've run it and the results look ok.  I'm not sure what we're going for here:

image

**Comment by sheryjoe** [comment]
@jadie1 @AtefehKashani when start_with_prepped_data tag is used, which distance transforms are used? The ones that are packaged/released with the use case or the previously groomed data run by the user on his local machine?

**Comment by jadie1** [comment]
@sheryjoe  When start_with_prepped_data is used it should use the distance transform released with the use case. It looks like that is the case for the Femur and the LA. But the Ellipse use case still calls the distance transform function when start_with_prepped_data is used, so that probably needs to be addressed.

**Comment by jadie1** [comment]
I added fixes for issues #593 and issue #667 and I tested the ellipsoid, femur, and left_atrium use cases.
I think the ellipsoid_fd use case needs to be updated still both in the python code and the data portal.

**Comment by jadie1** [comment]
@riddhishb Could you update the ellipsoid fixed domain use case on this branch to address issue #614 both in ellipsoid_fd.py and on the data portal? You can look at the ellipsoid use case to see how the folders have been renamed/restructured.

**Review by jadie1 — COMMENTED** [review]
I believe the naming is now consistent across all use cases and I verified that all use cases are running as expected. @sheryjoe I think this is ready to merge.

**Comment by sheryjoe** [comment]
@jadie1  Let's merge. Also, these changes should propagate to new use cases 
- lumps and femur_mesh @medakk
- sphere and femur with cutting planes @HeavenlyBerserker

**Comment by akenmorris** [comment]
@jadie1 Can you help resolve this conflict?

<<<<<<< usecases
    meshFiles = meshFiles[:15]
    print(meshFiles)

    pointDir = parentDir + 'shape_models/'

=======
    pointDir = parentDir + 'PointFiles/'
>>>>>>> master


Is it PointFiles or shape_models?

**Comment by jadie1** [comment]
It should be shape_models in all use cases now. Likewise it should be
"groomed" rather than "PrepOutput" now. This is consistent with the new
portal data.

**Comment by cchriste** [comment]
Can this be merged now?

**Comment by 0leks** [comment]
Question: were all of the dataset versioning issues caused by the switch PrepOutput to groomed and PointFiles to models?

**Comment by jadie1** [comment]
@0leks I'm not exactly sure which issues you are talking about but I think all current issues in master will be resolved when this gets merged in.

**Pull request merged**